app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListFragment.java

```diff
@@ -208,4 +224,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == SPEECH_REQUEST_CODE) {
+            String searchInput = SpeechUtil.processSpeechResult(resultCode, data, getActivity());
+            if (searchInput != null)
```

Comment:
nit: Please add the brackets for consistency.
```suggestion
            if (searchInput != null) {
```

Reply:
Okey, I usually remove it to get less number of code lines.

Reply:
Less code is good, but I feel like this trick makes the code look more like Python than Java. Also, the rest of the codebase is already using the brackets; it's best to be consistent.